### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by haywardt** [nominalista/expenses#6]
It's awesome. This would be awesomer with:
Tasker integration so my bank transaction sms messages could go in automatically.

If you displayed my average cash burn rate for the last 30 days, 90 days, 365 days. This would make budgeting really easy.

**Comment by nominalista** [comment]
Hello

Let me understand your idea better. When you make some bank transaction you receive the SMS confirmation and then it would be great to automatically add the expense to the app, am I right?

About the statistics - I plan to do so. I will let you know when it will be ready.

**Comment by haywardt** [comment]
Yes. I understand it is difficult to get SMS permissions through Google
play so I thought Tasker might serve as a bridge (they have SMS access but
are one of the few apps where it is allowed)

Here is an example message:

From:REGBNK
Account *1008: Transaction of $4.00 was received.
History:LAST
Balance:BAL
More info:INFO

**Comment by gcandrade10** [comment]
Hi, Hayward I just read this feature request and I am implementing it because I'm interested too.
My first approach will be with keywords, if the message contains the keyword an empty expense will be created with the message as comment and you will be able to edit it with the correct amount.

Next step will be recognize the amount and I see 2 options:
1. User can create a rule to find the amount given the keyword
2. Use Machine learning to find the amount

What do you think? Are you still interested in this feature? any suggestion?

**Comment by haywardt** [comment]
I have had good luck scraping US websites by looking for the $999.99 and I
think this would work here.

**Comment by gcandrade10** [comment]
That's a fine approach but since I want that to fit all currencies I created that in a rule way, you
 can check it here if you want:
https://github.com/gcandrade10/expenses/tree/feature/sms
https://github.com/gcandrade10/expenses/releases
